ksqldb-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -895,9 +895,7 @@
         {"topic": "right_topic", "key": 3, "value": {"L2": "d"}, "timestamp": 60000}
       ],
       "outputs": [
-        {"topic": "LEFT_JOIN", "key": 0, "value": {"L1": "A", "L2": null}, "timestamp": 0},
         {"topic": "LEFT_JOIN", "key": 0, "value": {"L1": "A", "L2": "a"}, "timestamp": 60000},
-        {"topic": "LEFT_JOIN", "key": 1, "value": {"L1": "B", "L2": null}, "timestamp": 330000},
```

Comment:
Can we add one more input record, to bump stream-time such that this left-join result will be eventually emitted?

Reply:
Added